File: parsePDF.js
const fs = require('fs');
const pdf = require('pdf-parse');

let dataBuffer = fs.readFileSync('Deliverable 1_ Scoping Report-3.pdf');

pdf(dataBuffer).then(function (data) {
    fs.writeFileSync('pdf_dump.txt', data.text);
    console.log('Dumped PDF text to pdf_dump.txt');
}).catch(e => console.error(e));

File: parse_pdf.py
import PyPDF2

try:
    with open('Deliverable 1_ Scoping Report-3.pdf', 'rb') as file:
        reader = PyPDF2.PdfReader(file)
        text = ''
        for page in reader.pages:
            text += page.extract_text() + '\n'
            
    with open('pdf_dump.txt', 'w', encoding='utf-8') as f:
        f.write(text)
    print("Done extracting PDF to pdf_dump.txt")
except Exception as e:
    print("Error:", e)
